components/automate-ui/src/app/modules/infra-proxy/create-databag-item-modal/create-databag-item-modal.component.spec.ts

```diff
@@ -0,0 +1,112 @@
+import { waitForAsync, ComponentFixture, TestBed } from '@angular/core/testing';
+import { ReactiveFormsModule } from '@angular/forms';
+import { StoreModule, Store } from '@ngrx/store';
+import { HttpErrorResponse } from '@angular/common/http';
+import { MockComponent } from 'ng2-mock-component';
+import { NgrxStateAtom, ngrxReducers, runtimeChecks } from 'app/ngrx.reducers';
+import { EventEmitter } from '@angular/core';
+import { HttpClient, HttpHandler } from '@angular/common/http';
+import { HttpStatus } from 'app/types/types';
+import { CreateDatabagItemModalComponent } from './create-databag-item-modal.component';
+import {
+  CreateDataBagItemSuccess,
+  CreateDataBagItemSuccessPayload,
+  CreateDataBagItemFailure
+} from 'app/entities/data-bags/data-bag-details.actions';
+import {
+  DataBagItem
+} from 'app/entities/data-bags/data-bags.model';
+
+describe('CreateDatabagItemModalComponent', () => {
+  let component: CreateDatabagItemModalComponent;
+  let fixture: ComponentFixture<CreateDatabagItemModalComponent>;
+
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        MockComponent({ selector: 'chef-modal', inputs: ['visible'] }),
+        MockComponent({ selector: 'chef-button', inputs: ['disabled'] }),
+        MockComponent({ selector: 'chef-error' }),
+        MockComponent({ selector: 'chef-form-field' }),
+        MockComponent({ selector: 'chef-loading-spinner' }),
+        MockComponent({ selector: 'chef-toolbar' }),
+        CreateDatabagItemModalComponent
+      ],
+      imports: [
+        ReactiveFormsModule,
+        StoreModule.forRoot(ngrxReducers, { runtimeChecks })
+      ],
+      providers: [
+        HttpClient,
+        HttpHandler
+      ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(CreateDatabagItemModalComponent);
+    component = fixture.componentInstance;
+    component.openEvent = new EventEmitter();
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
```

Comment:
Could we add actual test here?

Reply:
@vivekshankar1 This is default test case added by Angular To check Component created or not. Please have a look other test cases, if I missed to add any test please let me know will add. Thank you.

Reply:
Yes correct @chaitali-mane , my bad this file actually has all the required tests. Thank you.